handlers/frame_change_pre.py

```diff
@@ -1,31 +1,49 @@
 import bpy
 from bpy.app.handlers import persistent
 from ..utils import node as utils_node
+from ..utils import openVDB_sequence_resolve_all
+from ..utils import clamp
 
 # Flag that saves us from having to iterate all node trees when no image sequences are used (the common case).
 # Set in image node export method, reset when new .blend is loaded in load_post.
 using_image_sequences = False
 
+# Flag that saves us from having to iterate all node trees when no image sequences are used (the common case).
+# Set in openVDB node export method, reset when new .blend is loaded in load_post.
+using_smoke_sequences = False
+
 
 @persistent
 def handler(scene):
     global using_image_sequences
-    if not using_image_sequences or scene.render.engine != "LUXCORE":
+    global using_smoke_sequences
+
+    if not (using_image_sequences or using_smoke_sequences) or scene.render.engine != "LUXCORE":
         return
 
-    found_sequence = False
+    found_image_sequence = False
+    found_smoke_sequence = False
     for mat in bpy.data.materials:
         if not mat.luxcore.node_tree:
             continue
 
+        if utils_node.find_nodes(mat.luxcore.node_tree, "LuxCoreNodeTexOpenVDB") or \
+                utils_node.find_nodes(mat.luxcore.node_tree, "LuxCoreNodeTexSmoke"):
+            found_smoke_sequence = True
+            for n in mat.luxcore.node_tree.nodes:
+                if n.name.split('.')[0] == "Material Output" and n.active:
+                    # Force a viewport update
+                    mat.luxcore.node_tree.links.new(n.inputs["Material"], n.inputs["Material"].links[0].from_socket)
```

Comment:
a) I don't like single-letter names, they make the code hard to read, please write out "node" in this case.

b) You should never make any decision based on `node.name`, it is a completely arbitrary name that can even be changed by the user in the UI. 
Use `node.bl_idname == "LuxCoreNodeMatOutput"` instead.

c) I do not understand why you search for an output node at all?
To force a viewport update, just use the same code as I did below: `mat.diffuse_color = mat.diffuse_color`

d) `break` from the loop after you forced the material update to save performance

Reply:
> c) I do not understand why you search for an output node at all?
> To force a viewport update, just use the same code as I did below: mat.diffuse_color = mat.diffuse_color

The openVDB node is used with volumes and null material which don't have a diffuse_color.

Reply:
Ok, but in the code above, it looks to me like you are editing the material's node tree to force an update, not some volume node tree. I have to admit I don't understand why directly editing the material wouldn't lead to the same result?